game/scripts/vscripts/gamemode.lua

```diff
@@ -140,6 +140,7 @@ function GameMode:OnGameInProgress()
   DebugPrint("[BAREBONES] The game has officially begun")
 
   -- initialize modules
+  InitModule(FilterManager)
```

Comment:
Should I also move the initialisation for the other modules that add Filters? `InProgress` only fires at minute 0 which leaves a few seconds after hero spawn where the Filters aren't actually in place.